src/_guides/language/language-tour.md

```diff
@@ -1096,8 +1097,59 @@ var loudify = (msg) => '!!! ${msg.toUpperCase()} !!!';
 assert(loudify('hello') == '!!! HELLO !!!');
 {% endprettify %}
 
+This example uses an anonymous function.
+More about those in the next section.
 
-### Lexical scope {#lexical-scope}
+### Anonymous functions
+
+Most functions are named, such as `main()` or `printElement()`.
+You can also create a nameless function,
+called an _anonymous function_.
```

Comment:
Calling it an "anonymous function" is fine.  I'd say something like "called an anonymous function, or sometimes a lambda or closure", since either of the latter two terms have become fairly widely used.